build.gradle.kts

```diff
@@ -11,7 +11,7 @@ plugins {
 
 allprojects {
   group = "uk.gov.justice.service.hmpps"
-  version = "0.0.6"
+  version = "0.1.0"
```

Comment:
This version should hopefully be going into the kotlin template so let's have a minor release. Not ready for a v1 yet though, we have more to do.